device-data/types/bolus/normal.md

```diff
@@ -57,7 +57,7 @@ This is the sub-type of `bolus` event that represents a bolus insulin dose deliv
 
 <!-- start normal -->
 
-The `normal` field encodes the numerical value of the dose of insulin delivered by an insulin pump. To avoid noise in the data (especially as some insulin pumps make it very easy to deliver "doses" of 0 units when no additional dose is recommended given the PWD's current blood glucose and insulin on board), we have chosen _not_ to allow the upload of boluses with a total delivered dose of 0 units; the value must be greater than 0.
+The `normal` field encodes the numerical value of the dose of insulin delivered by an insulin pump. To avoid noise in the data (especially as some insulin pumps make it very easy to deliver "doses" of 0 units when no additional dose is recommended given the PWD's current blood glucose and insulin on board), we have chosen _not_ to allow the upload of boluses with a total delivered dose of 0 units; the value must be greater than 0. Note that it is still possible for a [`dual/square`](dual-square.md) bolus that has been cancelled to have a total delivered dose of 0 units.
```

Comment:
If the bolus was cancelled (i.e, there is an `expectedNormal`) we'll upload it as long as it's not the same as `normal`, for all sub-types. What I was trying to make clear here (and failed) is that a combo bolus can have a `normal` of 0 units with no `expectedNormal` and still pass, as long as there's something in the extended portion. (combo bolus doc refers to this section for normal portion).